config/default.json

```diff
@@ -0,0 +1,3 @@
+{
+    "apiKey":"EAco9tkerMBqitin1irWhCDipOxYHxJ5A5xe1nqQ"
```

Comment:
Just the info, you don't have to change anything.
In general api keys shouldn't be exposed to public repo. We could use `.env.local` file to store such keys but we won't be able to do anything to hide api calls when we'll deploy the website. Ideally this should be handled by an external server but we can't do that.

Reply:
I know, I've tried  that using config package, but I found out that this package is only for backend.